mapper-pojo/src/main/java/org/hibernate/search/v6poc/entity/pojo/mapping/PojoMappingContributor.java

```diff
@@ -6,8 +6,6 @@
  */
 package org.hibernate.search.v6poc.entity.pojo.mapping;
 
-import java.util.Set;
```

Comment:
Don't we have a checkstyle rule for that? Or we haven't enabled checkstyle yet?

Reply:
Mmmh, maybe I missed something but when I reviewed, I just saw this import being removed, without any code removal. I reviewed commit per commit so maybe it's because of that.